static/js/src/latest-news.js

```diff
@@ -0,0 +1,18 @@
+import {fetchLatestNews} from '@canonical/latest-news';
+
+// Pull latest news from blog feed
+const articlesContainerSelector = '#latest-news-container';
+const articleTemplateSelector = '#articles-template';
+
+const articlesContainer = document.querySelector(articlesContainerSelector);
+const articlesTemplates = document.querySelector(articleTemplateSelector);
+
+if (articlesContainer && articlesTemplates) {
+  fetchLatestNews({
+    articlesContainerSelector,
+    articleTemplateSelector,
+    limit: 2,
+    id: 1286,
```

Comment:
You didn't change the key to the previous comment.

Reply:
@anthonydillon You asked for it to be changed to `1286`.

<img width="492" alt="Screenshot 2020-10-28 at 21 38 34" src="https://user-images.githubusercontent.com/505570/97499544-f358b380-1965-11eb-9fc5-0974591d0d82.png">

Reply:
Oh, missed that.